Docs

kspieks opened the pull request:
This PR creates a starting point for the T3 documentation. Let me know how you want to handle the version number (I referred to this as v1.0.0). Looking forward to your feedback!

alongd commented on docs/source/contribute.rst:
```diff
@@ -0,0 +1,18 @@
+as a benefit to the community in the hopes that others may find it useful as well.
+
+We welcome contributions from the community. Please see the `contributor guidelines`__
+on T3's GitHub page, as well as the Code of Conduct.
```
could you link to the code of conduct file?

alongd commented on docs/source/index.rst:
```diff
@@ -0,0 +1,40 @@
+T3 - RMG-ARC Tandem Tool v\ |release|
```
without "The", it's not T3 ('The RMG-ARC Tandem Tool")

alongd commented on docs/source/release.rst:
```diff
@@ -0,0 +1,23 @@
+Release notes
+=============
+
+T3 1.0.0
```
our current version should be 0.1.0

alongd commented on docs/source/release.rst:
```diff
@@ -0,0 +1,23 @@
+T3 1.0.0
+^^^^^^^^^
+
+This is the first stable version of T3. See the API.
```
0.1.0 is not "stable"

alongd commented on docs/source/cite.rst:
```diff
@@ -0,0 +1,10 @@
+
+Text form::
+
+    version 1.0.0, https://github.com/ReactionMechanismGenerator/T3
```
should add authors and the repository name

alongd commented on docs/source/links.txt:
```diff
@@ -0,0 +1,21 @@
+.. _roadmap: https://github.com/ReactionMechanismGenerator/T3/wiki/Roadmap
+.. _semantic: https://semver.org/
+.. _SMILES: https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system
+.. _truhlar: https://dx.doi.org/10.1016/j.cpc.2016.09.004
```
Are all these links being used in out docs? let's leave only the ones that are

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,110 @@
+=========================
+
+Note: T3 was only tested on Linux (Ubuntu_ 18.04.1 LTS) and Mac machines. We don't expect it to work on Windows.
+T3 can be installed on a server, as well as on your local desktop / laptop, submitting jobs to the server/s.
```
T3 doesn't require RMG to be installed, it's a conda dependency

kspieks replied:
My bad. I didn’t pay attention to the environment file

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,110 @@
+
+Note: T3 was only tested on Linux (Ubuntu_ 18.04.1 LTS) and Mac machines. We don't expect it to work on Windows.
+T3 can be installed on a server, as well as on your local desktop / laptop, submitting jobs to the server/s.
+Note that T3 requires installation of both `RMG <http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html>`_, Reaction Mechanism Generator, and `ARC <https://reactionmechanismgenerator.github.io/ARC/installation.html>`_, Automated Rate calculator.
```
note that ARC should be installed on the same machine T3 is installed on

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,110 @@
+Install dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+- Install the latest DEVELOPER version of RMG (which has Arkane).
```
no need to install RMG for T3 (though the user may do so - perhaps we should have two env files, with and w/o `rmgpy`)

kspieks replied:
I think including RMG in the environment file made sense. I just didn't pay attention before...

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,110 @@
+Add these to your ``.bashrc`` file, which can be edited by typing, e.g., ``nanoe ~./bashrc``)::
+	
+	export t3_path=$HOME`/Path/to/T3'
+	alias t3_env='source activate t3_env'
```
let's recommend `t3e` instead

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,110 @@
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some optional, but convenient, aliases are listed below (make sure to change `~/Path/to/T3/` accordingly).
```
here we have `~/`, below we don't
although it's a dummy, let's be consistent

kspieks replied:
I made them all include `~/` now. Good catch thanks!

kspieks commented:
Thanks for reviewing this PR! I incorporated your feedback by squashing it into my previous commits. Please let me know if I should edit anything else :)

alongd commented on docs/source/index.rst:
```diff
@@ -0,0 +1,40 @@
+.. image:: t3.png
+   :align: center
+
+**T3 - RMG-ARC Tandem Tool** is a software for automating 
```
should be:
`**T3 - The RMG-ARC Tandem Tool** is a software for automating`

alongd commented on docs/source/release.rst:
```diff
@@ -0,0 +1,23 @@
+=============
+
+T3 0.1.0
+^^^^^^^^^
```
these `^^^` markings must be the same length as the text above them

alongd commented on docs/source/release.rst:
```diff
@@ -0,0 +1,23 @@
+
+This is the first version of T3.
+
+T3 is an RMG-ARC Tandem Tool for automated kinetic model generation and refinement.
```
I think:
"T3 is a tool for automated kinetic model generation and refinement."

alongd commented on tandem/utils.py:
```diff
@@ -5,6 +5,7 @@
 import os
 from typing import Optional, Union
 
+VERSION='0.1.0'
```
please add spaces before and after the `=` operator

alongd commented on docs/source/cite.rst:
```diff
@@ -0,0 +1,10 @@
+
+Text form::
+
+    A. Grinberg Dana, H. Wu, X. Dong, K.A. Spiekermann, W.H. Green, The RMG-ARC Tandem Tool (T3), version 0.1.0, https://github.com/ReactionMechanismGenerator/T3
```
I suggest:
" A. Grinberg Dana, K.A. Spiekermann, X. Dong, H. Wu, W.H. Green, The RMG-ARC Tandem Tool (T3) for automated kinetic model generation and refinement, version 0.1.0, source code: https://github.com/ReactionMechanismGenerator/T3"

alongd commented on docs/source/credits.rst:
```diff
@@ -0,0 +1,19 @@
+
+__ greengp_
+
+T3's contributors are:
```
please change the order to align with the citation

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,106 @@
+
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
please take a look at all the `^^^` line lengths

kspieks commented:
Thanks for reviewing and for your close attention to detail (sorry to miss some things). I'll update this tomorrow

kspieks commented:
I incorporated your feedback by squashing it into my previous commits. Let me know if I can add/edit anything else :)

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,106 @@
+
+- Download and install the `Anaconda Python Platform`__ for Python 3.7 or higher if you haven't already.
+- Get git if you don't have it already by typing sudo apt-get install git in a terminal.
+- Clone T3's repository to by typing the following command in the desired folder (e.g., under `~/home/Code/`)::
```
remove extra `to`

kspieks replied:
My bad. Thanks for catching that

alongd commented on docs/source/installation.rst:
```diff
@@ -0,0 +1,106 @@
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some optional, but convenient, aliases are listed below (make sure to change `~/Path/to/T3/` accordingly).
+Add these to your ``.bashrc`` file, which can be edited by typing, e.g., ``nanoe ~./bashrc``)::
```
I think it should be `nano`, not `nanoe`

kspieks replied:
Indeed, another sloppy typo on my end..

alongd commented on docs/source/cite.rst:
```diff
@@ -0,0 +1,8 @@
+
+Text form::
+
+    A. Grinberg Dana, K.A. Spiekermann, X. Dong, H. Wu, W.H. Green, The RMG-ARC Tandem Tool (T3) for automated kinetic model generation and refinement, version 0.1.0, source code: https://github.com/ReactionMechanismGenerator/T3
```
Can we add some reasonable line breaks for better readability?

kspieks replied:
Added some line breaks!

alongd commented:
I got the following warning when compiling locally: `source/links.txt:2: WARNING: Duplicate explicit target name: "arc".` Could you take a look?

Also, do you think we should add the API docs in this PR, or in a separate one? (should be relatively easy, I think)

kspieks commented:
Fixed the warning! I had mistakenly defined a link for ARC in installation.rst as well as in links.txt which had caused the warning.

I added the API to this PR and also added output and running rst files. Thanks so much for reviewing!

kspieks commented:
Of course! Τhe comments were helpful, and I appreciate your feedback!
